pilot/pkg/config/kube/crdclient/client.go

```diff
@@ -359,6 +365,7 @@ func (cl *Client) addCRD(name string) {
 	if s.IsBuiltin() {
 		kc = kclient.NewUntypedInformer(cl.client, gvr, filter)
 	} else {
+		cl.logger.Infof("delayedInformer for %v", gvr)
```

Comment:
Is this needed or is it a leftover of debugging?

Reply:
Leftover. Thanks for spotting that and I will remove it.